vsts/vsts.yaml

```diff
@@ -244,55 +289,57 @@ jobs:
       - script: |
```

Comment:
I think you should update the hosted agent to be same as what we have in main -> win 2022